GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -1707,13 +1758,107 @@ private void writeStatsAndGraphsSheet(
         }
     }
 
-    /**
-     * creates a cell in the row passed in and sets the style and value (if non-null)
-     */
+    private int writeCollectionStats(Map<QuestionGroupDto,
+            List<QuestionDto>> questionMap,
+            Sheet sheet,
+            int curRow) {
+        final int tagCol = 0;
+        final int valCol = 3;
+        //Global statistics
+        //Calculate them first (due to the window, we cannot sum this while looping and 
+        // then go back up and draw it last)
+        int totalQuestions = 0;
+        for (List<QuestionDto> group : questionMap.values()) {
+            totalQuestions += group.size();
+        }
+        //Now draw them
+        curRow++;
+        Row statRow = getRow(curRow++, sheet);
+        createCell(statRow, tagCol, "Questions");
+        createCell(statRow, valCol, totalQuestions);
+   
+        statRow = getRow(curRow++, sheet);
+        createCell(statRow, tagCol, "Form submissions");
+        createCell(statRow, valCol, totalInstances);
+        
+        if (totalInstances == 0) return curRow + 2; //add a little space
```

Comment:
Maybe it is just me but I usually avoid leaving out {} even with one line. Maybe flow conventions are diferent though.

Reply:
It's ok if on one line according to the last style guide i read, but it reminds me that Emmanuel likes it on a separate row, too...